app/features/send/cashu-send-quote-hooks.ts

```diff
@@ -625,8 +626,6 @@ export function useProcessCashuSendQuoteTasks() {
             });
           }
         });
-
-      await cashuSendService.markSendQuoteAsPending(sendQuote);
```

Comment:
This was the reason why the app would crash when you would try to pay already paid invoice. I don't remeber why I added this here but doesn't make sense because we also have regular onPending handler. It also doesn't make sense to mix it with initiate mutation because intiaite could pass and this fail and then you have retries of initiate too.
I played with sends and all still seems to be working fine so seems safe to remove